dev/com.ibm.ws.kernel.feature/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -21,7 +21,7 @@
 		</AD> 
         <AD id="feature" name="%feature.name" description="%feature.desc" required="false" type="String" cardinality="2147483647"/>
         <AD id="package.server.conflict" name="internal" description="internal use only" 
-            type="Boolean" default="false" required="false"/>
+            type="String" required="false"/>
```

Comment:
Would the code be clearer if this was a required property with a default of "false"?

Reply:
This is a hidden property that is internal.  Do we ever make internal properties required?

Reply:
i don't know ... but i didn't think that was an issue if there was a default value?

Reply:
I can put back `default="false"`, but I don't think I should change required to true.